.circleci/config.yml

```diff
@@ -565,7 +565,7 @@ workflows:
           matrix:
             parameters:
               edition: ["ee"]
-              java-version: ["java-11"]
+              java-version: ["java-17"]
```

Comment:
This feels like a big change. Is this why some of the automated tests aren't passing?